src/main/java/com/quantumlytangled/gravekeeper/GraveKeeperConfig.java

```diff
@@ -214,9 +218,9 @@ public static void loadConfig(final File file) {
         .get("soulbound", "soulbound_enchantment_names", SOULBOUND_ENCHANTMENT_NAMES,
             "List of enchantment names that are considered as soulbinding")
         .getStringList();
-    SOULBOUND_TAG_BOOLEAN = config
-        .get("soulbound", "soulbound_tag_boolean", SOULBOUND_TAG_BOOLEAN,
-            "List of NBT boolean that are considered as soulbinding when set to true")
+    SOULBOUND_TAG_VALUE = config
```

Comment:
* it's a tag list, consider renaming the key and variables accordingly
SOULBOUND_TAG_VALUE -> SOULBOUND_TAG_VALUES
soulbound_tag_value -> soulbound_tag_values
* We're still only supporting boolean tags, this is not explicit in the new description.